### DISCUSSION TITLE
Payment Issue

### DISCUSSION THREAD

**Discussion by KS441-ctrl** [community/community#86525] [New to GitHub]
I have tried subscribing for copilot it says my account is currently blocked from purchases so i want to unlock it how can i do that and i dont have second credit card so i was not even able to with current one so revert back pls to this message .
Thank You

**Comment by martinwoodward** [comment]
For[ information on adding, editing and removing payment details](https://docs.github.com/en/billing/managing-your-github-billing-settings/adding-or-editing-a-payment-method) see [here](https://docs.github.com/en/billing/managing-your-github-billing-settings/adding-or-editing-a-payment-method). 

However in your instance you might need to [contact support directly](https://support.github.com/contact) so that can get the information from you to help you with your enquiry.  As a reminder, please never post any personal information in this public forum.
